packages/console/src/ds-components/Button/index.tsx

```diff
@@ -122,25 +125,42 @@ export function LinkButton({
   type = 'default',
   size = 'medium',
   className,
+  href,
   targetBlank,
   ...rest
 }: LinkProps) {
-  return (
-    <a
-      className={classNames(styles.button, styles[type], styles[size], className)}
-      {...rest}
-      {...(Boolean(targetBlank) && {
+  const { getTo } = useTenantPathname();
+  const props = useMemo(
+    () => ({
+      ...rest,
+      className: classNames(styles.button, styles[type], styles[size], className),
+      ...(Boolean(targetBlank) && {
```

Comment:
Can we simply pop up the target and ref props to the outer scope?

Reply:
Will refactor Button component in a separated task.
https://linear.app/silverhand/issue/LOG-7011/refactor-button-component-in-admin-console